docs/getting-started/github-apps/github-apps.rst

```diff
@@ -6,6 +6,13 @@ Automatic Syncing with GitHub Apps and .dockstore.yml
 
 Overview
 --------
+Summary of GitHub App usage:
+
+
+#. Install Dockstore GitHub App
```

Comment:
A  link to how to "Install Dockstore GitHub App" on the line that says that would be useful for someone like me.